CoffeeEngine/src/CoffeeEngine/Audio/Audio.h

```diff
@@ -44,6 +44,8 @@ namespace Coffee {
     {
     public:
 
+        static constexpr const wchar_t* DefaultAudioPath = AKTEXT("assets/audio/Wwise Project/GeneratedSoundBanks/Windows");
```

Comment:
This does not compile in linux, in gcc it should be a const char pointer. We have to test the suggested changes in msvc also.

```suggestion
        static constexpr const char* DefaultAudioPath = "assets/audio/Wwise Project/GeneratedSoundBanks/Windows";
```